Unreal Engine plugin documentation update (#6350)
getsentry/sentry-docs: src/platforms/common/enriching-events/context.mdx

</PlatformSection>

<PlatformSection supported={["unreal"]}>

<Note>

> Sentry SDK automatically attaches certain context values (i.e. `Engine version`, `Configuration`, `Target Type`, etc.) to all events.
> 

Reviewer: ```suggestion
Sentry's SDK automatically attaches certain context values such as `Engine version`, `Configuration`, and `Target Type`, to name a few, to all events.

```
